config/01-ce-router-defaults.conf

```diff
@@ -151,89 +150,83 @@ JOB_ROUTER_TRANSFORM_OrigRequests @=jrt
 JOB_ROUTER_TRANSFORM_Cpus @=jrt
     # Outside of the HTCondor WholeNodeJobs case, set RequestCpus to one of the following, in order:
     # 1. 'xcount' from the source job
-    # 2. RequestCpus from the source job
+    # 2. RequestCpus from the source job, but only if > 1
     # 3. default_xcount from the job route
     # 4. 1
+
+    if defined MY.xcount
+        EVALSET RequestCpus xcount
+    else
+        EVALMACRO test_RequestCpus ($(MY.RequestCpus:0)) > 1
+        if $(test_RequestCpus)
+            # RequestCpus already correct
+        else
+            EVALSET RequestCpus $(default_xcount:1)
+        endif
+    endif
+
+    # the old-style transforms ignored RequestCPUs for... reasons? when setting these. we do the same here.
+    EVALSET remote_SMPGranularity  xcount ?: ($(default_xcount:1))
+    COPY    remote_SMPGranularity  remote_NodeNumber
+
+    SET JobIsRunning           (JobStatus =!= 1) && (JobStatus =!= 5)
+
     if $(test_want_whole_node)
-        SET JOB_GLIDEIN_Cpus       "$$(MY.TotalCpus ?: JobCpus)"
+        SET JOB_GLIDEIN_Cpus       "$$([MY.TotalCpus ?: JobCpus])"
```

Comment:
What's the purpose of adding the square brackets here?

Reply:
RTM.  $$() syntax is $$(attribute) or $$([expression])

Reply:
For anyone reading this in the future, the documentation @johnkn is referring to can be found in [condor_submit](https://htcondor.readthedocs.io/en/latest/man-pages/condor_submit.html#index-363), which is perhaps not the first place someone would look for if they're looking for information about job router configuration or ClassAd transforms.